trp_use_case/model/use_case.py

```diff
@@ -143,6 +143,12 @@ class use_case_collection(models.Model):
     _name = 'use_case.collection'
     _description = 'Set of use cases'
 
+    x_status = fields.Selection([
```

Comment:
the `x_` prefix is (by convention) reserved for manually added fields via the UI. Exactly in the case of the state field, you suffer a double penalty because if the field is called only `state`, you get some functionality for free (see below)